tutorials/spring-boot-restful/src/main/kotlin/org/jetbrains/kotlin/demo/Application.kt

```diff
@@ -0,0 +1,13 @@
+package org.jetbrains.kotlin.demo
+
+import org.springframework.boot.SpringApplication
+import org.springframework.boot.autoconfigure.SpringBootApplication
+
+@SpringBootApplication
+class Application {
```

Comment:
Ok, I have update the example with no empty body, looks nicer and I guess people will find easily how to add it if needed.